lib/generators/install/install_generator.rb

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 require 'rails/generators/active_record'
-class InstallGenerator < ActiveRecord::Generators::Base 
+class InstallGenerator < ActiveRecord::Generators::Base
   source_root File.expand_path('../templates', __FILE__)
 
   def setup_initializer
     template 'mse_router_info.yml', 'config/mse_router_info.yml'
     copy_file 'microservices_engine.rb',
               'config/initializers/microservices_engine.rb'
-    generate 'migration', 'CreateMicroservicesEngineConnection name:string object:string url:string'
+    # generate 'migration', 'CreateMicroservicesEngineConnection name:string object:string url:string'
```

Comment:
Did you comment this out for CI reasons? If so, we should restore it so it doesn't cause problems with #46.